src/scss/tokens/_screens.scss

```diff
@@ -1,7 +1,7 @@
 $screen-sm: 640px;
 $screen-md: 768px;
 $screen-lg: 1024px;
-$screen-xl: 1280px;
+$screen-xl: 1134px;
```

Comment:
What's the reason for changing this?

Reply:
@tillywoodfield the new design has a max width of `1134px` and `$page-width-max` is set to `$screen-xl` in [`_spacing.scss`](https://github.com/IATI/design-system/blob/main/src/scss/tokens/_spacing.scss).